src/main/java/com/stripe/param/EventListParams.java

```diff
@@ -242,18 +296,21 @@ public Builder addAllType(List<String> elements) {
     @SerializedName("lte")
     Long lte;
 
-    private Created(Long gt, Long gte, Long lt, Long lte) {
+    private Created(Map<String, Object> extraParams, Long gt, Long gte, Long lt, Long lte) {
+      this.extraParams = extraParams;
       this.gt = gt;
       this.gte = gte;
       this.lt = lt;
       this.lte = lte;
     }
 
     public static Builder builder() {
-      return new com.stripe.param.EventListParams.Created.Builder();
+      return new Builder();
```

Comment:
I don't fully grasp this change and if it can cause any issue so flagging

Reply:
It's the usual fully-qualified vs. short name. We're in a class named `Builder`, so  `Builder` refers to the class itself.